core/jazz_login/config/dev-config.json

```diff
@@ -1,4 +1,5 @@
 {
   "USER_POOL_ID":"{conf-user-pool-id}",
-  "CLIENT_ID":"{conf-client-id}"
+  "CLIENT_ID":"{conf-client-id}",
+  "ADMIN_ID": "{jazz_admin}"
```

Comment:
Include {jazz_admin} variable for jazz_login in service-configuration-data-loader file.

Reply:
done